conans/test/command/download_test.py

```diff
@@ -0,0 +1,79 @@
+import unittest
+import os
+from conans.model.manifest import FileTreeManifest
+from conans.model.ref import ConanFileReference, PackageReference
+from conans.paths import CONAN_MANIFEST, CONANINFO
+from conans.test.utils.tools import TestClient, TestServer
+from conans.util.files import save
+
+
+class DownloadTest(unittest.TestCase):
+
+    def download_reference_without_packages_test(self):
+        server = TestServer()
+        servers = {"default": server}
+
+        client = TestClient()
+        client = TestClient(servers=servers, users={"default": [("lasote", "mypass")]})
+        conanfile = """from conans import ConanFile
+class Pkg(ConanFile):
+    name = "pkg"
+    version = "0.1"
+"""
+        client.save({"conanfile.py": conanfile})
+        client.run("export . lasote/stable")
+
+        ref = ConanFileReference.loads("pkg/0.1@lasote/stable")
+        self.assertTrue(os.path.exists(client.paths.conanfile(ref)))
+
+        client.run("upload pkg/0.1@lasote/stable")
+        client.run("remove pkg/0.1@lasote/stable -f")
+        self.assertFalse(os.path.exists(client.paths.export(ref)))
+
+        client.run("download pkg/0.1@lasote/stable", ignore_error=True)
```

Comment:
If you are using ``ignore_error`` please capture the return of client and check that it is True (error = True)

Reply:
``ignore_error=True`` was not necessary indded. Thanks